main/global_state.h

```diff
@@ -50,6 +51,8 @@ typedef struct
     work_queue stratum_queue;
     work_queue ASIC_jobs_queue;
 
+    char * device_name;
```

Comment:
this has been replaced by device_model and device_model_str in the latest master of official repo